packages/block-library/src/buttons/style.scss

```diff
@@ -1,6 +1,7 @@
 // Increased specificity to override blocks default margin.
 .wp-block-buttons .wp-block-button {
 	display: inline-block;
+	margin-left: 0;
```

Comment:
i think some worked happened on master around margins, should we just keep this about the light DOM?

Reply:
Yeah, I don't think that change is supposed to be part of this PR. I think it may have been caused by a mistake during a rebase. I'll remove it. :+1:

Reply:
Actually, now that I think about it, it might be necessary to prevent the Button blocks from using the default block margins (which they might not have been getting before due to the wrapper `div`s or something). I'll try the PR without this line locally, and then if it works I'll push the change.

Reply:
Yeah, it turns out it's a necessary change to prevent this from happening with `aligncenter`-ed Buttons blocks:
![image](https://user-images.githubusercontent.com/19592990/85598784-87f68200-b611-11ea-822c-03bebb25e41e.png)

Reply:
I'll move the line to the `editor.scss` so it only applies in the editor, though, since the block editor auto-margins are only applied there.

Reply:
#23222